plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -259,7 +273,12 @@ public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, MetadataEn
         }
     }
 
-    private List<AddFileEntry> loadActiveFiles(TableSnapshot tableSnapshot, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, ConnectorSession session)
+    private List<AddFileEntry> loadActiveFiles(
+            TableSnapshot tableSnapshot,
+            TupleDomain<DeltaLakeColumnHandle> partitionConstraint,
```

Comment:
move partitionConstraint after metadataEntry, protocolEntry -- metadataEntry and protocolEntry are constant for given table (table-specific), while partitionConstraint is query-specific.